examples/fishjam-chat/package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "video-chat-example",
+  "name": "fishjam-chat-example",
```

Let's use similar approach as in `web-client-sdk` and have:
```suggestion
  "name": "@fishjam-example/fishjam-chat",
```